modules/cpmso-yaml-failure-domain-azure.adoc

```diff
@@ -8,17 +8,21 @@
 
 The control plane machine set concept of a failure domain is analogous to existing Azure concept of an link:https://learn.microsoft.com/en-us/azure/azure-web-pubsub/concept-availability-zones[_Azure availability zone_]. The `ControlPlaneMachineSet` CR spreads control plane machines across multiple failure domains when possible.
 
-When configuring Azure failure domains in the control plane machine set, you must specify the availability zone name.
+When configuring Azure failure domains in the control plane machine set, you must specify the availability zone name. An Azure cluster uses a single subnet that spans multiple zones.
 
 .Sample Azure failure domain values
```

Comment:
@RadekManak can you confirm which versions of OCP this format (including a `subnet` field, even though it's just one subnet) impacts? I think only 4.15+, but want to make sure.